jobs/monitor_jobs.groovy

```diff
@@ -0,0 +1,153 @@
+import utilities.StatusUpdater
+
+dirs = [
+  [name: 'grafana', slackChannel: 'monitor'],
+  [name: 'influxdb', slackChannel: 'monitor'],
+  [name: 'telegraf', slackChannel: 'monitor'],
+]
+
+dirs.each { Map dir ->
+  dir.commitEnvVar = "${dir.name.toUpperCase().replaceAll('-', '_')}_SHA"
+}
+
+dirs.each { Map dir ->
+  [
+    // for each repo, create a <dir.name>-master and <dir.name>-pr job
+    [type: 'master', branch: 'master'],
+    [type: 'pr', branch: '${sha1}'],
+  ].each { Map config ->
+    isMaster = config.type == 'master'
+    isPR = config.type == 'pr'
+
+    name = isMaster ? dir.name : "${dir.name}-pr"
+    downstreamJobName = defaults.testJob[config.type]
```

Comment:
Do you want to kick off downstream test job?  (would be `workflow-test(-pr)` running workflow e2e suite)

If there are no tests covering these yet, I'd see we remove the downstream job for now to shore up resources... we can always (and should) add back when tests are added.